docker/Dockerfile-x86_64

```diff
@@ -1,5 +1,5 @@
-FROM centos:5.11
-MAINTAINER The ManyLinux project
+FROM centos:6.8
```

Comment:
The base image is important per https://www.python.org/dev/peps/pep-0513/#the-manylinux1-policy in which several versions of common libraries are assumed.  If you switch to a different version of CentOS there may be some additional followup work to revert some of those libs.